scripts/gui/microbe_editor.mjs

```diff
@@ -241,26 +252,28 @@ function onSelectNewOrganelle(organelle){
     }
 }
 
-//! Updates the GUI buttons based on selected organelle
+//! Updates the GUI buttons based on selected organelle and button status
 function updateSelectedOrganelle(organelle){
```

Comment:
I just realized something. Why does this function need to be changed? The user is blocked from selecting a disabled organelle in the click callback. This function should work like it was before.

Reply:
that function isn't called in else branch of if(common.isInEngine) so with some test (maybe done bad) i suppose that updateOrganelle was called every iteration with GUI.